packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -28,36 +29,38 @@ export class BoosterGraphQLDispatcher {
 
     this.graphQLSchema = new GraphQLGenerator(config, commandDispatcher, this.readModelDispatcher).generateSchema()
     this.websocketHandler = new GraphQLWebsocketHandler(
+      config,
       logger,
-      this.config.provider.readModels.notifySubscription.bind(null, config),
+      this.config.provider.auth,
+      this.config.provider.connections,
       {
         onStartOperation: this.runGraphQLOperation.bind(this),
         onStopOperation: this.readModelDispatcher.unsubscribe.bind(this.readModelDispatcher),
-        onTerminateOperations: this.readModelDispatcher.unsubscribeAll.bind(this.readModelDispatcher),
+        onTerminate: this.handleDisconnect.bind(this),
       }
     )
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public async dispatch(request: any): Promise<any> {
-    const envelope = await this.config.provider.graphQL.rawToEnvelope(request, this.logger)
-    this.logger.debug('Received the following GraphQL envelope: ', envelope)
+    const envelopeOrError = await this.config.provider.graphQL.rawToEnvelope(request, this.logger)
```

Comment:
I will explain this `envelope or error` thingy below, in the `graphql-atapter.ts` file